deploy/charts/istio-operator/templates/operator-rbac.yaml

```diff
@@ -365,6 +365,15 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
this might need to be added to the controller.go file as well to generate it for the kustomize manifests